deployments/main.tf

```diff
@@ -0,0 +1,95 @@
+variable "aiven_api_token" {
+  type = string
+}
+
+variable "aiven_project_name" {
+  type = string
+}
+
+variable "aiven_service_name" {
+  type = string
+}
+
+variable "aiven_service_plan" {
+  type = string
+}
+
+variable "aiven_kafka_cloud_name" {
+  type = string
+}
+
+variable "aiven_kafka_topic_name" {
+  type = string
+}
+
+variable "aiven_kafka_username" {
+  type = string
+}
+
+terraform {
+  required_providers {
+    aiven = {
+      source  = "aiven/aiven"
+      version = "2.1.19"
+    }
+  }
+}
```

Comment:
Moving this in a `providers.tf` file would help to separate terraform plugin (providers) dependencies from the main code

Reply:
Since time ago, I see the best practice is to include all the providers inside the `main.tf`, especially when the file is not big.